docs/2.0/docs/accountfactory/guides/vend-aws-account.md

```diff
@@ -0,0 +1,49 @@
+# Using the Account Factory Workflow
+
+## Introduction
+
+The Account Factory Workflow in your `infrastructure-live-root` repository can be used to vend new AWS accounts. It takes a single input consisting of a JSON payload that you can generate from the `account-factory-inputs.html` web page.
+
+We use a JSON payload to support more vending opitions, as GitHub Workflows are limited to only 10 inputs.
+
+:::note
+This guide pertains only to non-delegated repositories. Enterprise customers are also able to [use Account Factory to create new Delegated Repositories](/2.0/docs/accountfactory/guides/delegated-repositories).
+:::
+
+### Step 1 - Download the file
+
+Navigate to your `infrastructure-live-root` repository and download the inputs web page located at `.github/workflows/account-factory-inputs.html` to your local machine.
+
+### Step 2 - Populate the values
+
+Open `account-factory-inputs.html` in a web browser and populate the input options.
+
+At the end of the page click "Generate", and copy the JSON output into your clipboard.
+
+### Step 3 - Run the Account Factory Workflow
+
+In GitHub, navigate to the Actions tab of your `infrastructure-live-root` repository and select `Account factory` on the left hand pane.
```

Comment:
Opportunity perhaps to generate links here using CustomizableValue and the customer's org/repo name?